rsmtool/modeler.py

```diff
@@ -1342,7 +1342,9 @@ def predict_train_and_test(self,
         config_as_dict = configuration.to_dict()
         config_as_dict.update(new_config_dict)
 
-        configuration = Configuration(config_as_dict, configuration.filepath)
+        configuration = Configuration(config_as_dict,
+                                      configdir=configuration.configdir,
+                                      filename=configuration._filename)
```

Comment:
This is a little weird to me. If we are going to access the private `_filename` attribute in such a public way then may be it should be a property too.

Reply:
Agree, let me make it a property. 
This line is necessary to make sure it's not lost when we re-instantiate the object.